2002-05-14  Rainer Orth  <ro@TechFak.Uni-Bielefeld.DE>

	* Make-lang.in (gnat1$(exeext), gnatbind$(exeext), gnattools):
	Restore $(CONFIG_H) and prefix.o dependencies.
	(ada.stage[1-4]): Depend on stage?-start.

--- gcc/ada/Make-lang.in.jj	2002-05-02 12:13:22.000000000 +0200
+++ gcc/ada/Make-lang.in	2002-12-09 17:45:36.000000000 +0100
@@ -595,16 +595,16 @@ ada.maintainer-clean:
 # Stage hooks:
 # The main makefile has already created stage?/ada
 
-ada.stage1:
+ada.stage1: stage1-start
 	-$(MV) ada/*$(objext) ada/*.ali ada/b_*.c stage1/ada
 	-$(MV) ada/stamp-* stage1/ada
-ada.stage2:
+ada.stage2: stage2-start
 	-$(MV) ada/*$(objext) ada/*.ali ada/b_*.c stage2/ada
 	-$(MV) ada/stamp-* stage2/ada
-ada.stage3:
+ada.stage3: stage3-start
 	-$(MV) ada/*$(objext) ada/*.ali ada/b_*.c stage3/ada
 	-$(MV) ada/stamp-* stage3/ada
-ada.stage4:
+ada.stage4: stage4-start
 	-$(MV) ada/*$(objext) ada/*.ali ada/b_*.c stage4/ada
 	-$(MV) ada/stamp-* stage4/ada
 
